sopel/bot.py

```diff
@@ -751,6 +753,18 @@ def search_url_callbacks(self, url):
 
 
 class SopelWrapper(object):
+    """Wrapper arround a Sopel instance and a Trigger
+
+    :param sopel: Sopel instance
+    :type sopel: :class:`sopel.bot.Sopel`
+    :param trigger: IRC Trigger line
+    :type trigger: :class:`sopel.trigger.Trigger`
+
+    This wrappers will be used to call Sopel's triggered commands and rules as
```

Comment:
Nemesi**s** strikes again!

```suggestion
    This wrapper will be used to call Sopel's triggered commands and rules as
```